cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/dstu3/Dstu3MeasureReportScorer.java

```diff
@@ -8,18 +9,60 @@
 import org.hl7.fhir.dstu3.model.MeasureReport.StratifierGroupComponent;
 import org.hl7.fhir.dstu3.model.MeasureReport.StratifierGroupPopulationComponent;
 import org.opencds.cqf.fhir.cr.measure.common.BaseMeasureReportScorer;
+import org.opencds.cqf.fhir.cr.measure.common.GroupDef;
 import org.opencds.cqf.fhir.cr.measure.common.MeasurePopulationType;
 import org.opencds.cqf.fhir.cr.measure.common.MeasureScoring;
+import org.opencds.cqf.fhir.cr.measure.common.PopulationDef;
 
 public class Dstu3MeasureReportScorer extends BaseMeasureReportScorer<MeasureReport> {
 
     @Override
-    public void score(MeasureScoring measureScoring, MeasureReport measureReport) {
+    public void score(Map<GroupDef, MeasureScoring> measureScoring, MeasureReport measureReport) {
         for (MeasureReportGroupComponent mrgc : measureReport.getGroup()) {
-            scoreGroup(measureScoring, mrgc);
+            scoreGroup(getGroupMeasureScoring(mrgc, measureScoring), mrgc);
         }
     }
 
+    protected MeasureScoring getGroupMeasureScoring(
+            MeasureReport.MeasureReportGroupComponent mrgc, Map<GroupDef, MeasureScoring> measureScoring) {
+        MeasureScoring measureScoringFromGroup = null;
+        // cycle through available Group Definitions to retrieve appropriate MeasureScoring
+        for (Map.Entry<GroupDef, MeasureScoring> entry : measureScoring.entrySet()) {
```

Comment:
issue: based on the fact there's no way for a dstu3 Measure to have group-level scoring (there's no support in the mapper component), there's no way for this code to be hit. Check out the code coverage report here:
https://app.codecov.io/gh/cqframework/clinical-reasoning/pull/391/blob/cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/dstu3/Dstu3MeasureReportScorer.java